library/tasks/ZarrUtils.wdl

```diff
@@ -0,0 +1,54 @@
+task SmartSeq2ZarrConversion {
+
+  #runtime values
+  String docker = "quay.io/humancellatlas/secondary-analysis-python3-scientific:0.1.6_zarr_test"
+  # the gene count file "<sample_id>_rsem.genes.results" in the task results folder call-RSEMExpression
+  File rsem_gene_results
+  # file named "<sample_id>_QCs.csv" in the folder  "call-GroupQCOutputs/glob-*" of the the SS2  output
+  Array[File] smartseq_qc_files
+  # name of the sample
+  String sample_name
+
+  meta {
+    description: "This  task will converts some of the outputs of Smart Seq 2 pipeline into a zarr file"
+  }
+
+  parameter_meta {
+    machine_mem_mb: "(optional) the amount of memory (MB) to provision for this task"
+    cpu: "(optional) the number of cpus to provision for this task"
+    disk: "(optional) the amount of disk space (GB) to provision for this task"
+    preemptible: "(optional) if non-zero, request a pre-emptible instance and allow for this number of preemptions before running the task on a non preemptible machine"
+    max_retries: "(optional) retry this number of times if task fails -- use with caution, see skylab README for details"
+  }
+
+  command {
+    set -euo pipefail
+
+    python3 /tools/create_zarr_ss2.py \
+       --qc_analysis_output_files_string ${sep=',' smartseq_qc_files} \
+       --rsem_genes_results  ${rsem_gene_results} \
+       --output_path_for_zarr  "${sample_name}.zarr" \
+       --sample_id ${sample_name} \
+       --format DirectoryStore
+
+    mkdir zarrout
+    #get all the files in the zarr folder in  a list
+    a=`find "${sample_name}.zarr"  -type f`
```

Comment:
In general, it's not a great idea to iterate over the output of `find`. See [here](https://mywiki.wooledge.org/BashPitfalls#for_f_in_.24.28ls_.2A.mp3.29) for a discussion.

Reply:
@mckinsel So do you have any preferences on this? (I might use some Python code here to do the iteration rather than bash `find`.)

Reply:
I am recommending that we merge this tested and functional feature into the greater system today. We will open a new ticket that captures this requested improvement for later releases. 

We have a solution to this comment written but it is untested. Merging the PR as it is now clears the path for other blocking PRs. Feel free to reach out to me directly for discussion.

Reply:
+1 to avoiding scope creep, please someone make said ticket in secondary-analysis and leave it in 'new issues' for me

Reply:
Yeah for sure, flag for later.